pseudocode.txt

```diff
@@ -19,14 +19,17 @@ Final file: tissue type subset where rows are the variants in the ranges of the
 
 
 
-TSV File indexed by individual, including TCGA-ID, Chrom, Pos, Ref, Alt, Genotype Info:
+TSV File organized by individual, where each variant has its own row including TCGA-ID, Chrom, Pos, Ref, Alt, Genotype Info:
 
     Loop through columns in tissue type subset (above)
         If genotype is not an empty field
             Add a row of TCGA-ID (from column), and also Chrom, Pos, Ref, Alt, Genotype Info to the final file
 
-Final file: TSV file indexed by individual, where each variant has its own row (with columns TCGA-ID, Chrom, Pos, Ref, Alt, Genotype Info)
-
+Final file: TSV file organized by individual, where each variant has its own row (with columns TCGA-ID, Chrom, Pos, Ref, Alt, Genotype Info)
```

Comment:
Here's what I was confused about earlier, the difference between the TSV file and the final file.  If the final file is what tells you which individuals have which variants, then you don't want the TCGA ID and genotype info in the TSV file (above).  You'd want the TSV file to be a nonredundant file that lists each of the unique variants in the cohort.